src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -340,6 +340,17 @@ private static IEnumerable<int> AdjustDimensions(OnnxShape shape)
             return new[] { 1 };
         }
 
+        /// <summary>
+        /// In the case that the ML.Net user wants a subset of columns or lists the columns in a different order then specified in the ONNX model,
+        /// we need to map from the ONNX model index to the ML.Net column index. This method does that mapping for us.
+        /// </summary>
+        /// <param name="iinfo">The index of the ML.Net column requested.</param>
+        /// <returns>The index o fht e</returns>
```

Comment:
Nit: Need to fix the return doc for this.

Reply:
Fixed.